release/example/en.custom/source/wordbreak.ts

```diff
@@ -0,0 +1,18 @@
+/*
+  Naively treat any character that is not a letter as a wordbreaking character.
+*/
+com.keyman.lexicalModel.registerWordBreaker('example.en.custom', function(text: string): Array<number> {
+  let regex = new RegExp(/[!a-zA-Z]/g);
+
+  let indexesOf = function(string) {
+    var match, indexes = [];
+
+    while (match = regex.exec(string)) {
+      indexes.push(match.index);
+    }
+
+    return indexes;
+  }
+
+  return indexesOf(text);
```

Comment:
Do you feel the indices in the string will be more useful than returning the actual words as strings? If word breaker functions return both the start and the end index of a word, then we can have it both ways: default is to return indices, and a simple function call can convert indices into words as strings.

Reply:
Good questions. Am happy to adjust this per the requirements of the wordbreaking when we get to it. I've defaulted to the most basic approach possible now of just marking the characters which are wordbreakers -- but I know that is naive. There are plenty of other wordbreak algorithms that we should research as well (yay!)